snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/request/SnomedConceptSearchRequestEvaluator.java

```diff
@@ -133,5 +114,43 @@ public Concepts evaluate(ResourceURI uri, ServiceProvider context, Options searc
 		);
 	}
 	
+	private Concept toConcept(ResourceURI codeSystem, SnomedConcept snomedConcept, String pt, boolean requestedExpand) {
+		final Concept concept = toConcept(codeSystem, snomedConcept, snomedConcept.getIconId(), pt, snomedConcept.getScore());
+		
+		SortedSet<Description> descriptions = generateGenericDescriptions(snomedConcept.getPreferredDescriptions());
```

Comment:
Why do we use only preferred descriptions here?

Reply:
Simplification, acceptable descriptions would require a smart join which is not that easy to do and most of the time only the preferred descriptions are needed.

Reply:
It just feels a little against the spirit of alternative terms to me to not use getDescriptions() here. 🙈